### ISSUE TITLE
Feature request: CalDAV support

### ISSUE THREAD

**Issue by paulequilibrio** [jendrikseipp/rednotebook#475]
It would be great if there was an option to sync notes with a CalDAV server using VJOURNAL.

**Comment by rogercreagh** [comment]
+1 definitely. I really want this.
Currently RedNoteBook is very limited as it is only a standalone programme. Adding VJOURNAL sync would enable sharing entries across devices. For Android phone/tablet there is an good Journal app using VJOURNAL (jtxBoard) and Nextcloud is capable of syncing VJOURNAL entries although there is no app (yet) to display them. Iit would be very useful to be able to add/edit entries from the desktop as well.

**Comment by jendrikseipp** [comment]
Great idea, both of you! If someone wants to work on adding support for VJOURNAL sync, I'd be happy to help.

**Comment by rogercreagh** [comment]
Well I'm just starting work on a component to sync VJOURNAL entries to/from a blog (initially Joomla,although a Wordpress one shouldn't be hard to follow on. Although I think WP has moved on a lot since I gave up using it).

For RedNoteBook the Python-Caldav project https://github.com/python-caldav/caldav is a client library for CalDav sync and probably would make a good starting point.

I don't currently speak Python but I'll certainly chip in any way I can...
